admin-ui/app/routes/License/LicenseDetailsPage.js

```diff
@@ -32,146 +24,140 @@ function LicenseDetailsPage({ item, loading, dispatch }) {
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="licenseKey"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                </Row>
-                <Row>
-                  <Col sm={6}>
-                    <GluuFormDetailRow
-                      label="fields.customerEmail"
-                      value={item.customerEmail}
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="customerEmail"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                  <Col sm={6}>
-                    <GluuFormDetailRow
-                      label="fields.customerName"
-                      value={
-                        item.customerFirstName + ' ' + item.customerLastName
-                      }
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="customerName"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                </Row>
-                <Row>
-                  <Col sm={6}>
-                    <GluuFormDetailRow
-                      label="fields.companyName"
-                      value={item.companyName}
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="companyName"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                  <Col sm={6}>
-                    <GluuFormDetailRow
-                      label="fields.validityPeriod"
-                      value={formatDate(item.validityPeriod)}
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="validityPeriod"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                </Row>
-                <Row>
-                  <Col sm={6}>
-                    <GluuFormDetailRow
-                      label="fields.isLicenseActive"
-                      value={item.licenseActive ? 'Yes' : 'No'}
-                      isBadge={true}
-                      lsize={3}
-                      rsize={9}
-                      doc_entry="isLicenseActive"
-                      doc_category={LICENSE}
-                    />
-                  </Col>
-                </Row>
-              </Container>
-            ) : (
-              <Alert severity="warning">
-                {!loading &&
-                  'The License API is not enabled for this application.'}
-              </Alert>
-            )}
-          </GluuLoader>
+          {item.licenseEnabled ? (
+            <Container style={{ backgroundColor: '#F5F5F5', float: 'left' }}>
```

Comment:
avoid using inline if it's possible, prefer to put the styling in another file `styles.js`

Reply:
I didn't change those lines. Since it contributes to improving the code quality, I will apply the feedback. Thanks!